crates/rpc/rpc-eth-api/src/helpers/transaction.rs

```diff
@@ -195,7 +195,7 @@ pub trait EthTransactions: LoadTransaction {
         &self,
         block_id: BlockId,
         index: usize,
-    ) -> impl Future<Output = Result<Option<Transaction>, Self::Error>> + Send
+    ) -> impl Future<Output = Result<Option<Transaction<Self::NetworkTypes>>, Self::Error>> + Send
```

Comment:
I see, this uses type aliases for syntax sugar, could we rename this from Transaction to RpcTransaction to make this more obvious